Firestore/core/test/unit/remote/bloom_filter_test.cc

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/remote/bloom_filter.h"
+#include <vector>
+#include "gtest/gtest.h"
+
+namespace firebase {
+namespace firestore {
+namespace remote {
+
+using util::Status;
+using util::StatusOr;
+
+class BloomFilterTest : public ::testing::Test {};
+
+TEST_F(BloomFilterTest, CanInstantiateEmptyBloomFilter) {
+  BloomFilter bloom_Filter = BloomFilter(std::vector<uint8_t>{}, 0, 0);
+  EXPECT_EQ(bloom_Filter.bit_count(), 0);
+}
+
+TEST_F(BloomFilterTest, CanInstantiateNonEmptyBloomFilter) {
+  {
+    BloomFilter bloom_Filter_1 = BloomFilter(std::vector<uint8_t>{1}, 0, 1);
```

Comment:
Remove the _1 and _2 suffixes from the "bloom_Filter" local variables. Since they are in different scopes their names will not collide and removing the suffixes improves readability. Please make this change here and elsewhere in this file.